playground/frontend/lib/modules/editor/parsers/run_options_parser.dart

```diff
@@ -16,7 +16,8 @@
  * limitations under the License.
  */
 
-RegExp regExp = RegExp(r'\-\-([A-z0-9]*)\s*([A-z0-9]*)\s*');
+RegExp pipelineOptionsRegExp =
+    RegExp(r"""--([A-z0-9]*)\s+([A-z0-9\/\"\'\*\-\:\;\.]*)""");
```

Comment:
nit: use single quotes instead for better readability => r'''--([A-z0-9]*)\s+([A-z0-9\/\"\'\*\-\:\;\.]*)'''

Reply:
Fixed, thanks